.github/actions/spell-check/expect.txt

```diff
@@ -1932,6 +1932,7 @@ timedate
 timediff
 timeunion
 timeutil
+titlebar
```

Comment:
This entry shouldn't be needed. There's an issue in some file. Let's remove it and the issue in the file too.